JsonFlatFileDataStore.Test/DataStoreTests.cs

```diff
@@ -493,6 +494,40 @@ public async Task FileContent_DefaultFormat(bool useLowerCamelCase)
 
             Assert.Contains(allowedLengths, i => i == content.Length);
             
+
+            UTHelpers.Down(path);
+        }
+
+        [Theory]
+        [InlineData(true, true)]
```

Comment:
We can add allowedLengths as a parameter, then no need to separate logic in the test

```cs
[InlineData(true, true, new [] { 40 } )]
```